include/inexor/vulkan-renderer/wrapper/device.hpp

```diff
@@ -80,6 +90,50 @@ class Device {
     [[nodiscard]] std::uint32_t get_transfer_queue_family_index() const {
         return transfer_queue_family_index;
     }
+
+#ifndef NDEBUG
+
+    /// @brief Vulkan debug marker: Sets the name of a Vulkan resource.
+    /// The debug marker name will be visible in external debuggers like RenderDoc.
+    /// @param object [in] A pointer to the Vulkan object.
+    /// @param type [in] The type of the Vulkan object.
+    /// @param name [in] The name of the debug marker which will be associated to the Vulkan object.
+    void set_object_name(const std::uint64_t object, const VkDebugReportObjectTypeEXT type, const char *name);
```

Comment:
Can't you stick to `const std::string &name`?

Reply:
You are right. I will fix this.

Reply:
Done.